src/supporting-data/citations/adapters/citationsConverter.ts

```diff
@@ -1,4 +1,4 @@
-export type Statistics = {
+export type LiteratureStatistics = {
```

Comment:
```suggestion
export type CitationsStatistics = {
```

Just to keep consistency with name of endpoint